extconf.rb

```diff
@@ -120,8 +120,15 @@ def rule(target, clean = nil)
 end
 
 case version_str = `#{pg_config} --version`
-when /^PostgreSQL ([7-9])\.([0-9]{1,3})(\.[0-9]{1,3})?$/
-   version = 10 * $1.to_i + $2.to_i
+when /^PostgreSQL ([7-9]|[1-9][0-9])\.([0-9]{1,3})(\.[0-9]{1,3})?$/
```

Comment:
This wont work on at least Debian (before or after your patch). My output is `"PostgreSQL 11.1 (Debian 11.1-1.pgdg90+1)\n"`.

Also the regex seems needlessly complicated and will break once we release 10.10. Why not something like: `/\APostgreSQL (\d+)\.(\d+)/`

We also may want to support development versions like the current "12devel".

Reply:
Thanks, fixed for the Debian case.  Support for development versions can be added later.